### PR TITLE
Make commands respect keyboard layout.

### PR DESCRIPTION
**Pull request by stephengeorgewest**
Fixes: #31

### CODE REVIEW

File: bambam.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
```
Comment (porridge) [inline bambam.py]: Please revert this change.

File: bambam.py [hunk 2]
```diff
@@ -171,8 +171,8 @@ def input(self, events, quit_pos):
             elif event.type == KEYDOWN or event.type == pygame.JOYBUTTONDOWN:
                 # check for words like quit
                 if event.type == KEYDOWN:
-                    if self.is_latin(event.key):
```
Comment (porridge) [inline bambam.py]: Since `self.is_latin` is now unused, please remove it.

Comment (stephengeorgewest) [reply bambam.py]: done

**Comment by porridge** [comment]
Thank you, please also fix checks which now fail, e.g. https://travis-ci.com/porridge/bambam/jobs/242528307 and I think this will be good to merge.
